static/less/group-detail.less

```diff
@@ -304,22 +304,14 @@ div.traceback > ul {
 
     &.is-expandable .title {
       cursor: pointer;
-
-      &:hover {
-        background: lighten(@blue-light, 36);
-      }
-    }
-
-    &.system-frame.is-expandable .title:hover {
-      background: darken(@white-dark, 5);
     }
 
     .title {
-      padding: 8px 20px;
+      padding: 6px 20px;
       font-size: 13px;
       margin: 0;
       line-height: 16px;
-      background: lighten(@blue-light, 35);
+      background: #faf9fb;
 
```

Comment:
I'm only doing this because I don't want to add anything to the variables.less file